src/locales/en/en.json

```diff
@@ -1186,8 +1186,8 @@
     "unableconnectwallet": "Unable to connect",
     "peeridsuccess": "Peer ID successful",
     "peeridnotrecognised": "Peer ID not recognised",
-    "disconnectwallet": "Disconnected successfully",
-    "strongbiometrynotavailable": "Strong biometry not available"
+    "strongbiometrynotavailable": "Strong biometry not available",
```

Comment:
Are we using `strongbiometrynotavailable`?

Reply:
No. I will remove it.

Reply:
And I have a question: on figma design, I found this alert after setup biometric failed. 
https://www.figma.com/design/XN6K76yt13zxCUHPh1Egr8/Identity-wallet?node-id=8622-13373&t=F2hCW94hiwkiAmhl-0

And in current source we got alert message when user cancel: 
<img width="721" alt="Screenshot 2024-05-24 at 10 04 59" src="https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/7fcb54f4-8a96-4bdb-9904-8ad9f88cdea3">

Are they same or I should create new alert for this message when we have another error ?

Reply:
Yes, it is the same.